pkgs/nsqio/nsq/registry.yaml

```diff
@@ -0,0 +1,29 @@
+packages:
+  - type: github_release
+    repo_owner: nsqio
+    repo_name: nsq
+    asset: nsq-{{trimV .Version}}.{{.OS}}-{{.Arch}}.go1.16.6.tar.gz
```

Comment:
Hmm.
go version `go1.16.6` is included.

Reply:
The version is changed per release.
https://github.com/nsqio/nsq/releases/tag/v1.2.0

```
nsq-1.2.0.darwin-amd64.go1.12.9.tar.gz
```

It is difficult for aqua to support this case.

Reply:
Sorry. This certainly complicates the configuration file.

Reply:
> But why include go version in assets name......

Yeah, I don't know.
It's really terrible. 😅